Gemfile

```diff
@@ -3,6 +3,7 @@ source 'https://rubygems.org'
 gem 'rails', '4.2.6'
 gem 'pg', '~> 0.15'
 gem 'sass-rails', '~> 5.0'
```

Comment:
Looks like I didn't get rid of all the unnecessary dependencies. I'll pare this down further once your changes are in master.